proto/substrait/algebra.proto

```diff
@@ -1164,4 +1163,10 @@ message AggregateFunction {
     // Use only distinct values in the aggregation calculation.
     AGGREGATION_INVOCATION_DISTINCT = 2;
   }
+
+  // This rel is used  to create references,
+  // in case we refer to a RelRoot field names will be ignored
+  message ReferenceRel {
```

Comment:
Doesn't this need to be added to the `oneof` in the `Rel` message?

Reply:
+1